UofT-DSI | pyhton - Assignment 2

olga-mineyeva opened the pull request:
## What changes are you trying to make? (e.g. Adding or removing code, refactoring existing code, adding reports)
completed the 3 sections of the assignment 2, Python module 

## What did you learn from the changes you have made?
reading files: passing a variable associated with a file into another variable (contents), which is of a _csv.reader data type

## Was there another approach you were thinking about making? If so, what approach(es) were you thinking of?
na

## Were there any challenges? If so, what issue(s) did you face? How did you overcome it?
In the section 2, it took me some time to understand how the return_values are integrated into the function being defined.
I figured out that return_values is a variable that must be assigned for all the subfunctions' results.

## How were these changes tested?
in a separate ipynb file with test arrays and with the test cells from the assignment file

## A reference to a related issue in your repository (if applicable)

## Checklist
- [x] I can confirm that my changes are working as intended

olga-mineyeva committed:
completed assignment_1

olga-mineyeva committed:
completed assignment_2

SidraBushra commented:
Hi, there is problem in your code, first where you have to find summary values using numpy library so you will use np.mean rather than data.mean, and in the last part where you have to do error detection in patient data, your function is incomplete, you have to use check_zeros function on the variable you assign patient_summary function, please complete your assignment till Sunday.

olga-mineyeva committed:
commit for assignment_2 resubmisison.md

olga-mineyeva commented:
Hi Sidra, thank you for your comment!
In the new commit:
1. Added a cell for all_paths reading all files and importing csv.
2. Corrected patient_summary function definition: corrected the code for mean values (np.mean() instead of data.mean()).
3. Ran a test for the check_zeros function.

SidraBushra commented:
Thanks, marked.
